denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc

```diff
@@ -15,90 +15,92 @@ tutorial_data:
   3:
     hologram:
       - "<&6>Rule <&ns>2"
-      - "<&e>Avoid profanity in public channels"
-      - "<&e>Keep your language kid appropriate"
+      - "<&e>Avoid profanity in public channels."
+      - "<&c>Keep your language kid appropriate."
     particle_guide: true
   4:
     hologram:
       - "<&6>Warp Command"
-      - "<&b>Warps can be made by any player, within their claim"
+      - "<&b>Warps can be made by any player within their claim."
       - "<&e>You can visit server warps, or other player's warps."
-      - "<&a>Command<&co> <&b>/warps"
+      - "<&a>Command Menu<&co> <&b>/warps"
     particle_guide: true
   5:
     hologram:
       - "<&6>Grim"
       - "<&e>Grim can help you return to your death location"
-      - "<&e>Be aware, nothing is free."
+      - "<&c>Be aware, nothing is free."
       - "<&a>--------------------------"
-      - "<&e>use <&b>/warps <&e>and go to the market."
+      - "<&e>Use <&b>/warps <&e>and go to the market to continue."
     particle_guide: true
   6:
     hologram:
       - "<&6>The Market"
-      - "<&e>You can buy all sorts of upgrades and items here"
+      - "<&e>You can buy all sorts of upgrades and items here!"
+      - "<&e>Be sure to check out the stocks and cosmetic vendors."
       - "<&e>You can also get daily rewards!"
     title: "<&6>Use <&b>/warps"
     subtitle: "Go to the server warp: Marketplace"
     message: "<&6>/warp &eto the server warp: Marketplace"
   7:
     hologram:
       - "<&6>Claims"
-      - "<&e>You can claim chunks to different groups"
-      - "<&e>If a group doesn't exist, it be created"
+      - "<&e>You can claim land!"
+      - "<&e>You can also give access to different groups of players"
+      - "<&e>If a group doesn't exist, it will be created"
       - "<&e>You can manage everything related to chunks in the GUI"
       - "<&a>Command<&co> <&b>/claims"
     particle_guide: true
   8:
     hologram:
       - "<&d>Soul Forge"
-      - "<&e>Here you can forge your items with souls"
-      - "<&e>Souls can be acquired in the <&c>Savage Lands"
+      - "<&e>Here you can forge your items with souls."
+      - "<&c>Souls can be acquired in the <&4>Savage Lands."
     particle_guide: true
   9:
     hologram:
-      - "<&7>The World"
-      - "<&e>The world is split into 2 sections"
-      - "<&e>The &cSavage Lands &eare the inner 20k"
-      - "<&e>The &2Outer Realms &eare everywhere outside of that"
+      - "<&d>The World (1)"
+      - "<&e>The world is split into 2 sections<&co>"
+      - "<&e>The &cSavage Lands &eare the inner 20,000 blocks".
+      - "<&e>The &2Outer Realms &eare everywhere outside of that."
     particle_guide: true
   10:
     hologram:
-      - "<&7>The World"
-      - "<&e>The <&c>Savage Lands &eare <&c>PvP Enabled"
-      - "<&e>Beware of your encounters here as mobs also spawn."
-      - "<&e>The <&2>Outer Realms <&e>are a safe haven"
-      - "<&e>No mobs will spawn, and PvP is disabled"
+      - "<&d>The World (2)"
```

Comment:
what's the (2)?

Reply:
There's two headings of The World, should i add Pt. 1 Pt. 2?

Reply:
Oh, didn't even notice the first, maybe `(1 / 2)` & `(2 / 2)`?

Reply:
Done.